packages/circuit-ui/components/ImageInput/ImageInput.tsx

```diff
@@ -223,12 +229,30 @@ const StyledLabel = styled(Label)<StyledLabelProps>(
   addButtonStyles,
 );
 
-const ActionButton = styled(IconButton)(
-  ({ theme }) => css`
-    position: absolute;
-    right: -${theme.spacings.bit};
-    bottom: -${theme.spacings.bit};
-  `,
+const actionButtonBaseStyles = ({ theme }: StyleProps) => css`
+  position: absolute;
+  right: -${theme.spacings.bit};
+  bottom: -${theme.spacings.bit};
+`;
+
+const actionButtonSizeStyles = ({ buttonSize }: ActionButtonProps) => {
+  if (buttonSize === 'giga') {
+    return css`
+      padding: 5px;
+      svg {
+        width: 14px;
+        height: 14px;
+      }
+    `;
+  }
+  return null;
+};
+
+type ActionButtonProps = IconButtonProps & { buttonSize: Size };
```

Comment:
Nitpick, we could make this `size` as well for consistency

Reply:
Ah no, because it would clash with the IconButton size prop. Hm. Let's keep it like this, it's internal anyways so would be easy to change without a breaking change.